src/test/java/core/mate/academy/service/MachineServiceTest.java

```diff
@@ -25,28 +25,28 @@ public static void beforeClass() {
     public void getAll_bulldozerType() {
```

Comment:
you mustn't modify `MachineServiceTest`, return it to its original state

Reply:
I took it from master branch